mbq/metrics/__init__.py

```diff
@@ -56,61 +57,59 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         pass
 
-    @property
-    def statsd(self):
-        return self._statsd or _statsd
```

Comment:
I just remembered why I used the property. The default collector (at the bottom of this file) is instantiated before `metrics.init` is called by the application so it has to _lazily_ fallback to the global `_namespace` at the time that it's being accessed. We'll need to do the same for the constant tags.

Reply:
I don't think we _need_ the properties, but I could be convinced that it's a nice to have